packages/antd/src/components/table/components/filterDropdown/index.tsx

```diff
@@ -1,12 +1,17 @@
-import React, { type ReactNode, useState } from "react";
-import { Button, Space } from "antd";
-import type { FilterDropdownProps as AntdFilterDropdownProps } from "antd/lib/table/interface";
+import React, { type ReactNode } from "react";
+import { Button, Space, TableColumnProps } from "antd";
 import dayjs from "dayjs";
 import { FilterOutlined } from "@ant-design/icons";
 import { useTranslate } from "@refinedev/core";
 
+type AntdFilterDropdownProps = React.ComponentProps<
+  Exclude<TableColumnProps<any>["filterDropdown"], ReactNode>
+>;
+
+export type MapValueEvent = "onChange" | "value";
+
 export type FilterDropdownProps = AntdFilterDropdownProps & {
-  mapValue?: (selectedKeys: React.Key[]) => any;
+  mapValue?: (selectedKeys: React.Key[], event: MapValueEvent) => any;
   children: ReactNode;
```

Comment:
```suggestion
  children: JSX.Element;
```

Can we also do this to enforce single child for `<FilterDropdown>`?